tools/translation-ai-prompts/pl.txt

```diff
@@ -72,26 +72,25 @@ Factions:
    - "tower" translates to "forteca"
 
 Units:
-   - "boars" translates to "dziki"
-   - "champions" translates to "czempioni"
-   - "deck of unit cards" translates to "talia jednostek"
-   - "dendroids" translates to "enty"
-   - "dread knights" translates to "rycerze grozy"
-   - "familiars" translates to "chowańce"
-   - "few" translates to "garstka"
-   - "griffins" translates to "gryfy"
-   - "griffins" translates to "gryfy"
-   - "hp" translates to "punkty zdrowia"
-   - "liches" translates to "licze"
-   - "marksmen" translates to "kusznicy"
-   - "neutral units" translates to "neutralne jednostki"
-   - "pack" translates to "grupa"
-   - "recruited or reinforced" translates to "zrekrutowane lub wzmocnione"
-   - "reinforce" translates to "wzmocnić"
-   - "reinforced" translates to "wzmocniony"
-   - "unit cards" translates to "karty jednostek"
-   - "units" translates to "jednostki"
-   - "zealots" tarnslates to "kapłani"
+   - "boars" translates to "[...]"
+   - "champions" translates to "[...]"
+   - "deck of unit cards" translates to "[...]"
+   - "dendroids" translates to "[...]"
+   - "dread knights" translates to "[...]"
+   - "familiars" translates to "[...]"
+   - "few" translates to "[...]"
+   - "griffins" translates to "[...]"
+   - "hp" translates to "[...]"
+   - "liches" translates to "[...]"
+   - "marksmen" translates to "[...]"
+   - "pack" translates to "[...]"
+   - "recruited or reinforced" translates to "[...]"
+   - "reinforce" translates to "[...]"
+   - "reinforced" translates to "[...]"
+   - "summoned unit" translates to "[...]"
+   - "unit cards" translates to "[...]"
+   - "units" translates to "[...]"
+   - "zealots" translates to "[...]"
```

Comment:
Coś tu poszło nie tak, zniknęły tłumaczenia 🤔

Reply:
Poza tym jest błąd. Nie "neutralne jednostki" a "jednostki neutralne".

Reply:
Nie wiem czemu, ale po uzupełnieniu prompt.txt, ta sekcja jest regenerowana po każdym ./build.sh pl Na razie wycofałem to z tego PR-a.

Reply:
Dzieje się tak dlatego, że artefakty wygenerowane przez po4a mamy w gitignore, one są generowane automatycznie każdorazowo przy wołaniu tego skryptu. Ten prompt jest wyjątkiem, bo chcemy go mieć gotowego do skopiowania. Zmiana w oryginalnym prompcie sprawia, że wpis w pliku po staje się _fuzzy_. Trzeba poprawić też plik `prompt.txt/pl.po`.